cypress/components/tabs/tabs.test.js

```diff
@@ -315,15 +315,14 @@ context("Testing Tabs component", () => {
     it.each(["error", "warning", "info"])(
       "should no longer report the any validation failures of children no longer mounted",
```

Comment:
```suggestion
      "should no longer report any validation failures of children no longer mounted",
```

We should probably fix the typo while updating the test anyway 😄